packages/p2p/src/components/app.jsx

```diff
@@ -58,10 +58,6 @@ const App = props => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    React.useEffect(() => {
-        setLanguage(lang);
-    }, [lang]);
```

Comment:
I removed the useEffect as there was another duplicate useEffect in the same file...